examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -5,9 +5,9 @@
 # Join token for the cluster.
 # Leave empty if the secret "teleport-kube-agent-join-token"
 # has been created before and contains a valid join token
-authToken: ""
+authToken: "test"
```

Comment:
I'm not sure we want to pre-fill these fields with default values here. This goes for all `authToken`, `proxyAddr`, `kubeClusterName` and `additionalLabels`. @webvictim what do you think?

Reply:
Agreed - these should be left blank so that Helm will error if you don't provide a required value.

Reply:
Agreed. That was a mistake on my end where i was testing it and forgot to scrub before committing :( Will fix